src/main/java/teammates/ui/controller/InstructorCourseEditPageAction.java

```diff
@@ -32,26 +32,26 @@ public ActionResult execute() throws EntityDoesNotExistException {
         
         /* Setup page data for 'Edit' page of a course for an instructor */
         InstructorCourseEditPageData data = new InstructorCourseEditPageData(account);
-        data.course = courseToEdit;
+        
+        List<InstructorAttributes> instructorList = new ArrayList<InstructorAttributes>();
+        int offset = -1;
```

Comment:
Should this be changed to `offset`?
Is it offsetting something or is it simply an index? (is it added on / subtracted from something somewhere?)
If `index` is more fitting, maybe change the `index` above to `indexStr` or something.

Also, a comment for why it is `-1` would be good. (e.g. "-1 for new instructor.", etc)